src/confluent_kafka/admin/__init__.py

```diff
@@ -809,3 +817,146 @@ def set_sasl_credentials(self, username, password):
         :raises TypeException: Invalid input.
         """
         super(AdminClient, self).set_sasl_credentials(username, password)
+
+    @staticmethod
+    def _make_describe_user_scram_credentials_result(f, futmap):
```

Comment:
All `_make_..._result` methods seems similar. Shall we make a generic method for all of these?`

Reply:
For the moment I've unified just these two